LBHFSSPublicAPI/Properties/launchSettings.json

```diff
@@ -23,7 +23,9 @@
         "DB_PORT": "5432",
         "DB_USERNAME": "postgres",
         "DB_PASSWORD": "mypassword",
-        "DB_DATABASE": "test-database"
+        "DB_DATABASE": "test-database",
+        "ADDRESSES_API_BASE_URL": "Test",
```

Comment:
Do we need to have variables in both the Tests **and** API project? I think adding them to the test project should suffice.

My worry here of adding these variables to the real project is that they are going to mask an issue. The idea behind throwing an exception from the startup, when ENV variable is not present is that whenever well try to launch an application from anywhere, it's going to notify us of this configuration error early.

By providing these dummy variables here, we're tricking our environment variable configuration check into thinking it's all configured, while it's actually not and will be the cause of a less intuitive error down the line.